openlibrary/core/vendors.py

```diff
@@ -150,8 +150,11 @@ def _get_amazon_metadata(id_, id_type='isbn'):
 
     if not lending.amazon_api:
         raise Exception("Open Library is not configured to access Amazon's API")
-    product = lending.amazon_api.lookup(**kwargs)
-    # sometimes more than one product can be returned, choose first
+    try:
+        product = lending.amazon_api.lookup(**kwargs)
+    except Exception as e:
```

Comment:
@mekarpeles the linting is complaining `e` is unused -- I think we should also catch the specific `HTTPError` -- not sure if we can reliably restrict it to 503, but splitting it to 503 for throttling, and handling other HTTPErrors seems to be what we need to deal with specifically.

Reply:
excepting Exception for now to be safe (removed the `e`)